tests/lib_test.php

```diff
@@ -219,55 +219,6 @@ public function test_get_links_student_view_reports() {
         $this->assertEquals('', $plagiarismturnitinsim->get_links($linkarray));
     }
 
-    /**
-     * Test that get_links returns a not sent message if the submission was not sent.
-     */
-    public function test_get_links_submission_not_sent() {
```

Comment:
This no longer needed?

Reply:
That's right. Not sent was used when the plugin was enabled after students had submitted.